sklearn/neighbors/_nearest_centroid.py

```diff
@@ -240,7 +292,7 @@ def predict(self, X):
         Parameters
         ----------
         X : {array-like, sparse matrix} of shape (n_samples, n_features)
-            Test samples.
+            Array of samples (test vectors).
```

Comment:
We can still predict on training data if we want. So let's be broad.
```suggestion
            Input data.
```

Reply:
Could you rename `C` to `y_pred`

Reply:
Sure!